ui/src/tasks/actions/v2/index.ts

```diff
@@ -468,9 +481,14 @@ export const getErrorMessage = (e: any) => {
   return message
 }
 
-export const getRuns = async (taskID: string): Promise<Run[]> => {
-  const runs = await client.tasks.getRunsByTaskID(taskID)
-  return runs
+export const getRuns = (taskID: string) => async (dispatch): Promise<void> => {
+  try {
+    const runs = await client.tasks.getRunsByTaskID(taskID)
+
+    dispatch(setRuns(runs))
+  } catch (error) {
+    console.error(error)
```

Comment:
Can this display a notification instead?

Reply:
Yup! I added a notification:)